Project.toml

```diff
@@ -4,11 +4,14 @@ repo = "https://github.com/JuliaDynamics/RecurrenceAnalysis.jl.git"
 version = "1.6.2"
 
 [deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 DelayEmbeddings = "5732040d-69e3-5649-938a-b6b4f237613f"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+DynamicalSystemsBase = "6e36e845-645a-534a-86f2-f5d4aa5a06b4"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+PyPlot = "d330b81b-6aea-500a-939a-2ce795aea3ee"
```

Comment:
None of these three have a reason to be dependencies. Especially PyPlot.

Reply:
Forgot about that one. Good point, thank you. Removed it.